src/apps/loan-list/modal/renew-loans-modal-content.tsx

```diff
@@ -34,6 +34,10 @@ const RenewLoansModalContent: FC<RenewLoansModalContentProps> = ({
   const { mutate } = useRenewLoansV2();
   const { close } = useModalButtonHandler();
   const { itemsShown, PagerComponent } = usePager(loansModal.length, pageSize);
+  // TODO: Investigate if we can use useIntersection instead of useInView
+  // and then we can remove the react-hook-inview dependency.
+  // useIntersection documentation:
+  // https://github.com/streamich/react-use/blob/master/docs/useIntersection.md#useintersection
   const [ref, isVisible] = useInView({ threshold: 0 });
```

Comment:
To me this is almost a requirement to add `react/use`.

I see `react/use` as a potentially single and well-maintained source of hooks for us going forward. If it cannot replace the other location were we use a third-party hook then the value already seems limited and we might as well go with more specialized implementations with a smaller surface according to our needs.

Reply:
Ok changed it...